src/OpenTelemetry.Exporter.Prometheus.HttpListener/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Added `target_info` to Prometheus exporters ([#5407](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5407))
```

Comment:
this is only if OpenMetrics format is requested?

Reply:
Yes, Prometheus will break on `info` types if you are returning text/plain instead of open metrics. Will update the changelog to be clearer.